localstack/services/events/provider.py

```diff
@@ -595,10 +593,33 @@ def filter_event_with_target_input_path(target: Dict, event: Dict) -> Dict:
     return event
 
 
+def process_event_with_input_transformer(target: Dict, event: Dict) -> Dict:
```

Comment:
nit: Consider adjusting the signature to facilitate re-use and testing:
`def process_event_with_input_transformer(input_transformer: Dict, event: Dict) -> Dict:`

Passing the `input_transformer` into this method to make it easier to re-use and in general follow the "Law of Demeter" (i.e., principle of least knowledge).